src/pages/non-pass-finalisation/cat-b/__tests__/non-pass-finalisation.cat-b.page.spec.ts

```diff
@@ -1,31 +1,31 @@
 import { ComponentFixture, async, TestBed } from '@angular/core/testing';
 import { IonicModule, NavController, Platform } from 'ionic-angular';
 import { NavControllerMock, PlatformMock } from 'ionic-mocks';
-import { AppModule } from '../../../app/app.module';
-import { AuthenticationProvider } from '../../../providers/authentication/authentication';
-import { AuthenticationProviderMock } from '../../../providers/authentication/__mocks__/authentication.mock';
+import { AppModule } from '../../../../app/app.module';
+import { AuthenticationProvider } from '../../../../providers/authentication/authentication';
+import { AuthenticationProviderMock } from '../../../../providers/authentication/__mocks__/authentication.mock';
 import { StoreModule, Store } from '@ngrx/store';
-import { StoreModel } from '../../../shared/models/store.model';
+import { StoreModel } from '../../../../shared/models/store.model';
 import { MockComponent } from 'ng-mocks';
-import { PracticeModeBanner } from '../../../components/common/practice-mode-banner/practice-mode-banner';
-import { NonPassFinalisationPage } from '../non-pass-finalisation';
-import { NonPassFinalisationViewDidEnter } from '../non-pass-finalisation.actions';
-import { ActivityCodeComponent } from '../../office/components/activity-code/activity-code';
-import { SetTestStatusWriteUp } from '../../../modules/tests/test-status/test-status.actions';
-import * as testActions from '../../../modules/tests/tests.actions';
+import { PracticeModeBanner } from '../../../../components/common/practice-mode-banner/practice-mode-banner';
+import { NonPassFinalisationCatBPage } from '../non-pass-finalisation.cat-b.page';
+import { NonPassFinalisationViewDidEnter } from '../../non-pass-finalisation.actions';
+import { ActivityCodeComponent } from '../../../office/components/activity-code/activity-code';
+import { SetTestStatusWriteUp } from '../../../../modules/tests/test-status/test-status.actions';
+import * as testActions from '../../../../modules/tests/tests.actions';
 import { TestFinalisationComponentsModule } from
-'../../../components/test-finalisation/test-finalisation-component.module';
-import { TestCategory } from '../../../shared/models/test-category';
+'../../../../components/test-finalisation/test-finalisation-component.module';
+import { TestCategory } from '../../../../shared/models/test-category';
 
 describe('NonPassFinalisationPage', () => {
```

Comment:
Can you update the describe string to include cat b

Reply:
Yes, thanks for catching this.